setup

```diff
@@ -143,9 +171,16 @@ def build():
     ):
         project_dir = os.path.join(RS_DIRECTORY, project)
         os.chdir(project_dir)
-        subprocess.run(["cargo", "build", "--release"])
-
-        release_dir = os.path.join(project_dir, "target", "release")
+        rs_build_mode = "release"
+        if args.type is not None:
+            if args.type == BuildType.DEBUG: rs_build_mode = "debug"
+            if args.type == BuildType.RELEASE: rs_build_mode = "release"
+        rs_build_flags = ["--release"]
```

Comment:
TODO(gitbuda): Append `--cargo-flags` from the `setup` script.

Reply:
Is this done?